django/db/models/expressions.py

```diff
@@ -138,6 +138,13 @@ def get_source_expressions(self):
     def set_source_expressions(self, exprs):
         assert len(exprs) == 0
 
+    def _parse_expressions(self, *expressions):
+        return [
+            arg if hasattr(arg, 'resolve_expression') else (
+                F(arg) if isinstance(arg, six.string_types) else Value(arg)
```

Comment:
No need for that. `text_type` is a subset of `string_types`.

Python 3:
```python
>>> six.text_type
<class 'str'>
>>> six.string_types
(<class 'str'>,)
```

Python 2:
```python
>>> six.text_type
<type 'unicode'>
>>> six.string_types
(<type 'basestring'>,)
>>> str.__bases__
(<type 'basestring'>,)
>>> unicode.__bases__
(<type 'basestring'>,)
```